yb-voyager/src/srcdb/oracle.go

```diff
@@ -444,3 +445,17 @@ func (ora *Oracle) GetColumnsWithSupportedTypes(tableList []*sqlname.SourceName,
 func (ora *Oracle) GetServers() []string {
 	return []string{ora.source.Host}
 }
+
+func (ora *Oracle) ClearMigrationState(migrationUUID uuid.UUID, exportDir string) error {
+	log.Infof("Clearing migration state for migration %q", migrationUUID)
+	log.Infof("Dropping table LOG_MINING_FLUSH")
+	_, err := ora.db.Exec("DROP TABLE LOG_MINING_FLUSH")
+	if err != nil {
+		if strings.Contains(err.Error(), "ORA-00942") {
+			// Table does not exist, so nothing to drop
+			return nil
+		}
+		return err
```

Comment:
Wrap the error with some context:
`return fmt.Errorf("drop table %s: %w", logMiningFlushTableName, err)`